cmd/terramate/cli/cloud_credential_google.go

```diff
@@ -504,7 +504,7 @@ func (g *googleCredential) Refresh() (err error) {
 		}()
 	}
 
-	const oidcTimeout = 3 // seconds
+	const oidcTimeout = 60 // seconds
```

Comment:
Why is this change on?

Reply:
this is unrelated to the feature but noticed that it was inconsistent. All other timeouts were updated to be 60s months ago and I forgot this one.
I can do in a separate PR though

Reply:
no worries, I thought it was a mistake :)